mantidimaging/gui/windows/operations/view.py

```diff
@@ -242,6 +243,8 @@ def roi_visualiser(self, roi_field):
             # Happens if nothing has been loaded, so do nothing as nothing can't be visualised
             return
 
+        self._roi_selector_window_count += 1
```

Comment:
We should probably prevent multiple ROI windows from opening, looks like it is a bit more work disable the button, so I'll make a new issue for this.

Reply:
I have opened #1602